src/sass/components/_header.scss

```diff
@@ -2,7 +2,7 @@
 // ======
 
 .global-header {
-  @include z-index('global-header');
+  z-index: $z-index-sticky;
```

Comment:
@mirisuzanne @stacyk This fixes a bug where the user menu dropdown (which has z-index 6000, but is within the `.global-header` at z-index 1) was showing up behind the path elements (z-index 5). Happy for you to implement a better fix.

![Screenshot 2020-03-17 16 19 41](https://user-images.githubusercontent.com/552316/76898967-15854180-686d-11ea-860b-e5b904994230.png)

Reply:
yep, that looks like the right approach.